plugins/inputs/logstash/logstash.go

```diff
@@ -290,6 +292,20 @@ func (logstash *Logstash) gatherPluginsStats(
 			return err
 		}
 		accumulator.AddFields("logstash_plugins", flattener.Fields, pluginTags)
+		if pluginType == "output" && plugin.Name == "elasticsearch" {
+			flattener := jsonParser.JSONFlattener{}
+			err := flattener.FlattenJSON("", plugin.BulkRequests)
+			if err != nil {
+				return err
+			}
+			accumulator.AddFields("logstash_plugins_bulk_requests", flattener.Fields, pluginTags)
+			flattener = jsonParser.JSONFlattener{}
+			err = flattener.FlattenJSON("", plugin.Documents)
+			if err != nil {
+				return err
+			}
+			accumulator.AddFields("logstash_plugins_documents", flattener.Fields, pluginTags)
+		}
```

Comment:
Hmmm why do you add two new metrics here instead of adding e.g. a tag denoting the plugin type (e.g. `plugin_type="documents"`)?

Reply:
It's a count of documents sent for the plugin type (`output`) that's already set.

Reply:
Ok, but why do you need to add two new _metrics_?

Reply:
I just put them where our internal scraper was putting them. You're right that it makes more sense to add them to the `logstash_plugins` metric.